src/components/TabsContent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { ListingItem } from "./ListingItem";
+import { useSelector } from "react-redux";
+import { ApplicationState } from "../app/store";
+import { ListItem } from "../types/ListItem";
+import { Tab } from "../types/Tab";
+
+export type TabsContentProps = {
+  displayTabs: Tab[];
+  selectedTabIndex: number;
+};
+export const TabsContent: (props: TabsContentProps) => JSX.Element = ({
+  displayTabs,
+  selectedTabIndex,
+}: TabsContentProps) => {
+  const { items } = useSelector((state: ApplicationState) => state);
+  return (
+    <div>
+      {displayTabs.map((tab) => {
+        let itemsToDisplay: ListItem[] = [];
```

Comment:
I think it might be nicer if the manipulation with `itemsToDisplay` variable is done outside the main `return` block. Just it seems a bit more standard.

Reply:
Good point, thanks!